scripts/prepare_for_tests.sh

```diff
@@ -4,7 +4,7 @@ set -e
 COMPILER_DIRECTORY="$(git rev-parse --show-toplevel)/crates/cast/tests/utils/compiler"
 CAIRO_REPO="https://github.com/starkware-libs/cairo/releases/download"
 
-SCARB_VERSIONS=("0.4.1" "0.5.2")
+SCARB_VERSIONS=("0.6.0")
```

Comment:
I'm not sure this will work, I had troubles updating to 0.6.0-rc1 (looked devnet did not support required libfuncs).
At the moment for sure it will fail, because hashes changed and a 2.0.2 compiler is required for devnet runner (they are set in tests/helpers/constants.rs). Scarb.toml test files also need compiler version bump in order to work.

You can try to run cast tests and see if it works with current devnet from master; alternatively you can revert this and I will try to bump it when new devnet releases.

Reply:
Forgot to link an issue https://github.com/foundry-rs/starknet-foundry/issues/429